provisioners/configure.sh

```diff
@@ -100,26 +102,50 @@ chown -R www-data /var/www/
 
 service apache2 stop
 a2dissite 000-default
-cp $TEMPLATES_PATH/etc/apache2/apache2.conf /etc/apache2/apache2.conf
-cp $TEMPLATES_PATH/etc/apache2/mods-enabled/mpm_prefork.conf /etc/apache2/mods-enabled/mpm_prefork.conf
-envsubst < $TEMPLATES_PATH/etc/apache2/sites-enabled/$PERM_SUBDOMAIN.permanent.conf > /etc/apache2/sites-enabled/$PERM_SUBDOMAIN.permanent.conf
-envsubst < $TEMPLATES_PATH/etc/apache2/sites-enabled/preload.permanent.conf > /etc/apache2/sites-enabled/preload.permanent.conf
-a2enmod expires
-a2enmod headers
-a2enmod rewrite
-a2enmod proxy
-a2enconf security
-a2enconf charset
-a2enconf other-vhosts-access-log
+cp "${TEMPLATES_PATH}/etc/apache2/conf-available/*" /etc/apache2/conf-available/
+envsubst \
+  < $TEMPLATES_PATH/etc/apache2/sites-available/$PERM_SUBDOMAIN.permanent.conf \
+  > /etc/apache2/sites-available/$PERM_SUBDOMAIN.permanent.conf
+envsubst \
+  < $TEMPLATES_PATH/etc/apache2/sites-available/preload.permanent.conf \
+  > /etc/apache2/sites-available/preload.permanent.conf
+a2ensite \
+  "${PERM_SUBDOMAIN}.permanent" \
+  preload.permanent
+a2enmod \
+  expires \
+  headers \
+  http2 \
```

Comment:
`mod_http2` is incompatible with `mpm_prefork`; now that we're using `mpm_event` we can use `mod_http2`. I confirmed I could make HTTP/2 requests via curl locally:

```sh
curl --http2 -v \
  --resolve dev.permanent.org:80:127.0.0.1 \
  --header "X-Forwarded-Proto: https" \
  --header 'Request-Version: 2' \
  http://dev.permanent.org/api/account/getAuthenticatedAccount
```

However, I think the AWS load balancer might need to be configured to make plaintext HTTP/2 requests, as it does not appear to be upgrading by default. Again, this is intended as a performance improvement, as it would theoretically allow the load balancer to pipeline more requests over a single connection and reduce connection overhead.

Reply:
Yes, correct! Loading it now makes Apache able to respond to HTTP/2 requests without affecting its ability to respond to HTTP/1.1 requests.